src/sqlparser/tests/sqlparser_common.rs

```diff
@@ -1968,9 +1968,9 @@ fn parse_literal_decimal() {
 
 #[test]
 fn parse_literal_string() {
-    let sql = "SELECT 'one', N'national string', X'deadBEEF'";
+    let sql = "SELECT 'one', N'national string', X'deadBEEF', E'c style escape string'";
```

Comment:
Although we have e2e sqllogictest, it is still better to make this test/example a little bit more representative.